Packs/GoogleThreatIntelligence/Integrations/GoogleThreatIntelligence/README.md

```diff
@@ -0,0 +1,2531 @@
+| GoogleThreatIntelligence.File.attributes.type_extension | String | Extension of the type. |
+| GoogleThreatIntelligence.File.attributes.tags | String | File tags. |
+| GoogleThreatIntelligence.File.attributes.last_analysis_date | Number | Last analysis date in epoch format. |
+| GoogleThreatIntelligence.File.attributes.unique_sources | Number | Unique sources. |
+| GoogleThreatIntelligence.File.attributes.first_submission_date | Number | First submission date in epoch format. |
+| GoogleThreatIntelligence.File.attributes.ssdeep | String | SSDeep hash of the file. |
+| GoogleThreatIntelligence.File.attributes.md5 | String | MD5 hash of the file. |
+| GoogleThreatIntelligence.File.attributes.sha1 | String | SHA-1 hash of the file. |
+| GoogleThreatIntelligence.File.attributes.magic | String | Identification of file by the magic number. |
+| GoogleThreatIntelligence.File.attributes.last_analysis_stats.harmless | Number | The number of engines that found the indicator to be harmless. |
+| GoogleThreatIntelligence.File.attributes.last_analysis_stats.type-unsupported | Number | The number of engines that found the indicator to be of type unsupported. |
+| GoogleThreatIntelligence.File.attributes.last_analysis_stats.suspicious | Number | The number of engines that found the indicator to be suspicious. |
+| GoogleThreatIntelligence.File.attributes.last_analysis_stats.confirmed-timeout | Number | The number of engines that confirmed the timeout of the indicator. |
+| GoogleThreatIntelligence.File.attributes.last_analysis_stats.timeout | Number | The number of engines that timed out for the indicator. |
+| GoogleThreatIntelligence.File.attributes.last_analysis_stats.failure | Number | The number of failed analysis engines. |
+| GoogleThreatIntelligence.File.attributes.last_analysis_stats.malicious | Number | The number of engines that found the indicator to be malicious. |
+| GoogleThreatIntelligence.File.attributes.last_analysis_stats.undetected | Number | The number of engines that could not detect the indicator. |
+| GoogleThreatIntelligence.File.attributes.meaningful_name | String | Meaningful name of the file. |
+| GoogleThreatIntelligence.File.attributes.reputation | Number | The reputation of the file. |
+| GoogleThreatIntelligence.File.type | String | Type of the indicator \(file\). |
+| GoogleThreatIntelligence.File.id | String | Type ID of the indicator. |
+| GoogleThreatIntelligence.File.links.self | String | Link to the response. |
+
+#### Command Example
+
+```!file file=6bcae8ceb7f8b3a503c321085d59d7441c2ae87220f7e7170fec91098d99bb7e```
+
+#### Context Example
+
+```json
+{
+    "DBotScore": {
+        "Indicator": "6bcae8ceb7f8b3a503c321085d59d7441c2ae87220f7e7170fec91098d99bb7e",
+        "Reliability": "A - Completely reliable",
+        "Score": 2,
+        "Type": "file",
+        "Vendor": "GoogleThreatIntelligence"
+    },
+    "File": {
+        "Extension": "txt",
+        "MD5": "bea65efcc00169dec4f7e2ed612e041f",
```

Comment:
Just verifying again, this is all dummy data?

Reply:
Yes but I edited them anyways.